### PR TITLE
Add Actions field to InstallationPermissions model

### PR DESCRIPTION
**Pull request by gsavit**
Addresses https://github.com/google/go-github/issues/1802

### CODE REVIEW

File: github/apps_test.go [hunk 1]
```diff
@@ -90,6 +90,7 @@ func TestAppsService_ListInstallations(t *testing.T) {
                                    "target_id":1,
                                    "target_type": "Organization",
                                    "permissions": {
+									   "actions": "read",
```
Comment (gmlewis) [inline github/apps_test.go:93]: Could you please make this line match the indentation of the following line (i.e. remove the tabs and replace with the same number of spaces in line 94)?

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @gsavit !
Just one minor tweak, please, and then I think we will be good to merge.

File: github/apps_test.go (hunk 1 above)
Comment (gsavit) [reply github/apps_test.go:93]: hmm yeh I noticed this, formatted multiple times and no change.. thought it might be weird gh rendering?

Comment (gsavit) [reply github/apps_test.go]: nevermind, fixed, sorry!

**Review by gmlewis — APPROVED** [review]
Thank you, @gsavit !
LGTM.
Merging after tests complete.

**Pull request merged**